package.json

```
@@ -102,5 +104,13 @@
       "javaPackageName": "com.reactcommunity.rndatetimepicker"
     }
   },
-  "packageManager": "yarn@4.1.1"
+  "peerDependencies": {
+    "react": "18.2.0",
```

```suggestion
    "react": "*",
```

let's relax this a bit